Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -123,6 +158,114 @@ namespace gdjs {
       return this._pixiObject.scale.y;
     }
 
+    /**
+     * @param x The layer column.
+     * @param y The layer row.
+     * @param layerIndex The layer index.
+     * @returns The tile's id.
+     */
+    getTileId(x: integer, y: integer, layerIndex: integer): integer {
+      const tileMap = this._tileMap;
+      if (!tileMap) return -1;
+      const layer = tileMap.getTileLayer(layerIndex);
+      if (!layer) return -1;
+      return layer.getTileId(x, y) || -1;
+    }
```

Comment:
Shouldn't this be in the object or the map model?
Ideally, the render should focus on rendering.

Reply:
Yes, good idea, I'll move this logic to the map model